src/Sounder.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2023 Matthew Taylor
+*/
+//
+
+#include "Sounder.h"
+#include "picoPlatform.h"
+#include "KiwiBoardFirmware_menu.h"
+
+PicoPlatform *hardware;
```

Comment:
This isn't getting set to the value of the platform object owned by the firmware main, so this is basically a null pointer, i'm honestly surprised it works at all?